### DISCUSSION TITLE
Support for replica-announce-ip and replica-announce-port

### DISCUSSION THREAD

**Discussion by djotanov** [apache/kvrocks#481] [Q&A] [answered]
In docker/kubernetes, instances don't have a fixed up. When working with dynamic IPs, sentinel will not work, unless kvrocks can be configured to announce DNS address of the instance (and mapped port if needed).
Does kvrocks support `replica-announce-ip' and 'replica-announce-port` properties (couldn't see them in config file)?
Redis support these since 3.2 version.

**Comment by git-hulk** [comment, marked as answer]
Kvrocks didn't support `replica-announce-ip` and `replica-announce-port` now, we can add it to the plan.
